pyro/distributions/transforms/affine_autoregressive.py

```diff
@@ -220,6 +223,110 @@ def _inverse_stable(self, y):
         return x
 
 
+@copy_docs_from(ConditionalTransformModule)
+class ConditionalAffineAutoregressive(ConditionalTransformModule):
+    """
```

Comment:
note: if you use [raw strings](https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals) you can replace all your double backslashes with single backslashes. Feel free to ignore
```py
r"""
:math:`\sigma`
"""
```

Reply:
Hang on a sec, I'll make this change

Reply:
done